cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -1274,6 +1275,7 @@ void trackSessionStart() {
   }
 
   public void updateServerSessionStart() {
+    isSessionStartCalled = true;
```

Comment:
Set to false here
https://github.com/cleverpush/cleverpush-android-sdk/blob/314507b02027c26853ee5ab1af993bb4fa64e041/cleverpush/src/main/java/com/cleverpush/CleverPush.java#L3477